src/components/organisms/AdminVenueView/components/RunTabUsers/RunTabUsers.tsx

```diff
@@ -1,60 +1,74 @@
 import React from "react";
+import { useParams } from "react-router-dom";
 import { useAsync } from "react-use";
 
-import { ALWAYS_EMPTY_ARRAY } from "settings";
+import { ALWAYS_EMPTY_ARRAY, STRING_PLUS } from "settings";
 
 import { getUserRef } from "api/profile";
 
-import { SpaceWithId } from "types/id";
 import { User } from "types/User";
 
 import { withId } from "utils/id";
 
+import { useWorldAndSpaceBySlug } from "hooks/spaces/useWorldAndSpaceBySlug";
 import { useShowHide } from "hooks/useShowHide";
 
 import { VenueOwnersModal } from "pages/Admin/VenueOwnersModal";
 
 import { RunTabUserInfo } from "components/organisms/AdminVenueView/components/RunTabUserInfo";
 
-import { ButtonNG } from "components/atoms/ButtonNG";
+import { AdminVenueViewRouteParams } from "../../AdminVenueView";
 
 import "./RunTabUsers.scss";
 
-interface RunTabSidebarProps {
-  space: SpaceWithId;
-}
+export const RunTabUsers: React.FC = () => {
+  const { worldSlug, spaceSlug } = useParams<AdminVenueViewRouteParams>();
 
-export const RunTabUsers: React.FC<RunTabSidebarProps> = ({ space }) => {
+  const { space, world } = useWorldAndSpaceBySlug(worldSlug, spaceSlug);
   const {
     isShown: isShownInviteAdminModal,
     show: showInviteAdminModal,
     hide: hideInviteAdminModal,
   } = useShowHide();
 
-  const owners = space.owners ?? ALWAYS_EMPTY_ARRAY;
+  const spaceEditors = space?.owners ?? ALWAYS_EMPTY_ARRAY;
+  const worldOwners = world?.owners ?? ALWAYS_EMPTY_ARRAY;
 
-  const { value: admins = ALWAYS_EMPTY_ARRAY } = useAsync(
+  const { value: editors = ALWAYS_EMPTY_ARRAY } = useAsync(
     async () =>
-      Promise.all(owners.map((owner) => getUserRef(owner).get())).then((docs) =>
-        docs.map((doc) => withId(doc.data() as User, doc.id))
-      ),
-    [owners]
+      Promise.all(
+        [...worldOwners, ...spaceEditors].map((admin) =>
+          getUserRef(admin).get()
+        )
+      ).then((docs) => docs.map((doc) => withId(doc.data() as User, doc.id))),
+    [spaceEditors, worldOwners]
   );
```

Comment:
could you put this function to the API directory with the name getUsersById where you can pass an array of userIds. I guess it'd useful.

Reply:
done [1c7dc50](https://github.com/sparkletown/sparkle/pull/2766/commits/1c7dc50efb7cb0990e2278c30e8f659cc06e5c78)